test/images/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+# the timeout was increased because some images might take longer to build (e.g.: sample-apiserver, pets/redis-installer)
+timeout: 3000s
+# this prevents errors if you don't use both _GIT_TAG and _PULL_BASE_REF,
+# or any new substitutions added in the future.
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20190906-745fed4'
+    entrypoint: make
+    env:
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+    - TAG=$_GIT_TAG
+    - BASE_REF=$_PULL_BASE_REF
+    - WHAT=$_WHAT
+    - REGISTRY=gcr.io/k8s-staging-e2e-test-images
+    args:
+    - all-push
```

Comment:
this doesn't exist
https://storage.googleapis.com/kubernetes-jenkins/logs/post-kubernetes-push-images/1216776522732408832/artifacts/build.log

Reply:
ahh https://github.com/kubernetes/kubernetes/pull/87162